rules/when-you-use-mentions-in-a-pbi/rule.md

```diff
@@ -39,7 +39,7 @@ The benefits of using comments are:
 
 
 
-When someone (especially the PO) asks you to fix a PBI, mention that person in the PBI comments so he knows when it’s fixed.
+When someone (especially the PO) asks you to fix a PBI, mention that person in the PBI comments so they knows when it’s fixed.
```

Comment:
Good change - "knows" should now be changed to "know".

Reply:
Good catch. Thanks!